### PR TITLE
Add license

### LINKED ISSUE #2: Add license

### PR DESCRIPTION
**Pull request by eganjs**
Resolves #2

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 UK Hydrographic Office
```
Comment (ciaranevans) [inline LICENSE]: Do we need to add the Crown Copyright? e.g: https://github.com/alphagov/frontend/blob/master/LICENCE.txt

Comment (eganjs) [reply LICENSE]: A great question, I do not believe so. This has been copied from https://github.com/UKHO/docs/blob/master/LICENSE.

Comment (ciaranevans) [reply LICENSE]: Have you asked? I doubt it needs commercial protection, but just in case...

Comment (eganjs) [reply LICENSE]: @MarcDenman thoughts?

Comment (MarcDenman) [reply LICENSE]: It can't hurt to add it in.

Comment (eganjs) [reply LICENSE]: Updated

**Pull request merged**